controllers/registry.go

```diff
@@ -502,10 +506,9 @@ func (r *VeleroReconciler) getSecretNameAndKey(credential *corev1.SecretKeySelec
 	return secretName, secretKey
 }
 
-func (r *VeleroReconciler) parseAWSSecret(secret corev1.Secret, secretKey string) (string, string, error) {
+func (r *VeleroReconciler) parseAWSSecret(secret corev1.Secret, secretKey string, matchProfile string) (string, string, error) {
 
-	AWSAccessKey, AWSSecretKey := "", ""
-	// this logic only supports single profile presence in the aws credentials file
+	AWSAccessKey, AWSSecretKey, AWSProfile := "", "", ""
 	splitString := strings.Split(string(secret.Data[secretKey]), "\n")
 	keyNameRegex, err := regexp.Compile(`\[.*\]`) //ignore lines such as [default]
```

Comment:
@deepakraj1997  NIT: The existing comment for this regex utility is misleading. Can you please remove/update this comment ?

Reply:
Yep, my bad, didnt notice this comment.